.travis.yml

```diff
@@ -14,7 +14,7 @@ before_install:
     if [[ "${TRAVIS_OS_NAME}" == "linux" ]]; then
     sudo apt-get install libwxgtk3.0-dev libgtk2.0-dev libjpeg62-dev libgcrypt20-dev liblzo2-dev
     CMAKE_URL="http://www.cmake.org/files/v3.13/cmake-3.13.0-Linux-x86_64.tar.gz"
-    mkdir -p ${TRAVIS_BUILD_DIR}/deps/cmake && travis_retry wget --quiet -O - ${CMAKE_URL} | tar --strip-components=1 -xz -C ${TRAVIS_BUILD_DIR}/deps/cmake
```

Comment:
You probably want to word the commit message to reference travis_retry, not travics-ci?

Reply:
sorry, I mistype the word.